manifests/m/MicrosoftCorporation/MicrosoftOffice/16.0.18129.20158/MicrosoftCorporation.MicrosoftOffice.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftOffice

```

```suggestion
PackageIdentifier: Microsoft.Office
```
Please update file name and path to match this change.